src/Carbon/Carbon.php

```diff
@@ -1227,8 +1227,13 @@ public function formatLocalized($format)
             $format = preg_replace('#(?<!%)((?:%%)*)%e#', '\1%#d', $format); // @codeCoverageIgnore
         }
 
+        $customToStringFormat = self::$toStringFormat;
+        self::resetToStringFormat();
+
         $formatted = strftime($format, strtotime($this));
```

Comment:
Why don't we just replace 
`$formatted = strftime($format, strtotime($this));` 
with 
`$formatted = strftime($format, $this->getTimestamp());`
?

Reply:
Very good point. Knowing `getTimestamp` makes the use of `strtotime` obsolete.

Reply:
See https://github.com/briannesbitt/Carbon/pull/1107#issuecomment-366469612